contracts/rewards/src/contract.rs

```diff
@@ -157,6 +157,12 @@ pub fn query(
                 .change_context(ContractError::SerializeResponse)
                 .map_err(axelar_wasm_std::ContractError::from)
         }
+        QueryMsg::VerifierParticipation { pool_id, epoch_num } => {
```

Comment:
How does the querier get to know the `epoch_num`? I thought it's internal and an implementation detail, no?

Reply:
It's optional, so normally won't be specified and will just default to current. But the query itself returns the current epoch, so you could query prior epochs if you want.